Superlista/Views/MainView.swift

```diff
@@ -132,7 +132,7 @@ struct MainView: View {
     func createNewListAction() {
         guard let user = dataService.user else { return }
         
-        let newOwner: OwnerModel = OwnerModel(id: user.id, name: user.name!)
+        let newOwner: OwnerModel = OwnerModel(id: user.id, name: user.name ?? "")
```

Comment:
Sempre que tiver uma "", se for um nome de lista coloca "NovaLista" (ou o localizable equivalente), e quando for um nome de usuário, chamar a função getNickname()